docs/developer-docs/daos/sns/managing/making-proposals.mdx

```diff
@@ -326,6 +326,81 @@ The SNS DAO has control over a treasury from which funds can be sent to other ac
 
 This is a ["critical" proposal type](#criticalproposals), which means it is subject to higher voting thresholds before it is accepted.
 
+#### Maximum 7-day Amount Total
+
+The rate that funds can be transferred from the treasury is limited. To explain how this works,
+first, define T to be the value of the tokens in the treasury in units of [XDR]. T can fall within
+three size regimes:
+
+[XDR]: https://en.wikipedia.org/wiki/Special_drawing_rights
+
+<table>
+  <thead>
+    <tr>
+      <th>Size</th>
+      <th>Max T (XDR)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>small</td>
+      <td>100_000</td>
+    </tr>
+    <tr>
+      <td>medium</td>
+      <td>1_200_000</td>
+    </tr>
+    <tr>
+      <td>large</td>
+      <td>(infinity)</td>
+    </tr>
+  </tbody>
+</table>
+
+The total amount that can be transferred from the treasury in a 7 day period is at most L(T), where
+L(T) is given by this table:
+
+<table>
+  <thead>
+    <tr>
+      <th>Treasury Size</th>
+      <th>L(T)</th>
+      <th>Fraction of T</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>small</td>
+      <td>T</td>
+      <td>100%</td>
+    </tr>
+    <tr>
+      <td>medium</td>
+      <td>0.25 * T</td>
+      <td>25%</td>
+    </tr>
+    <tr>
+      <td>large</td>
+      <td>300_000 XDR</td>
+      <td>0% - 25%</td>
+    </tr>
```

Comment:
I don't quite understand what the two columns in the table mean. Is it the larger of the two or are these just two alternative ways how to state the same maximum?